docs/src/pages/premium-themes/onepirate/modules/views/ProductHowItWorks.tsx

```diff
@@ -1,133 +1,122 @@
 import * as React from 'react';
 import { Theme } from '@material-ui/core/styles';
-import { withStyles, WithStyles } from '@material-ui/styles';
+import { SxProps } from '@material-ui/system';
+import Box from '@material-ui/core/Box';
 import Grid from '@material-ui/core/Grid';
 import Container from '@material-ui/core/Container';
 import Button from '../components/Button';
 import Typography from '../components/Typography';
 
-const styles = (theme: Theme) =>
-  ({
-    root: {
-      display: 'flex',
-      backgroundColor: theme.palette.secondary.light,
-      overflow: 'hidden',
-    },
-    container: {
-      marginTop: theme.spacing(10),
-      marginBottom: theme.spacing(15),
-      position: 'relative',
-      display: 'flex',
-      flexDirection: 'column',
-      alignItems: 'center',
-    },
-    item: {
-      display: 'flex',
-      flexDirection: 'column',
-      alignItems: 'center',
-      padding: theme.spacing(0, 5),
-    },
-    title: {
-      marginBottom: theme.spacing(14),
-    },
-    number: {
-      fontSize: 24,
-      fontFamily: theme.typography.fontFamily,
-      color: theme.palette.secondary.main,
-      fontWeight: theme.typography.fontWeightMedium,
-    },
-    image: {
-      height: 55,
-      marginTop: theme.spacing(4),
-      marginBottom: theme.spacing(4),
-    },
-    curvyLines: {
-      pointerEvents: 'none',
-      position: 'absolute',
-      top: -180,
-      opacity: 0.7,
-    },
-    button: {
-      marginTop: theme.spacing(8),
-    },
-  } as const);
+const item: SxProps<Theme> = {
```

Comment:
`SxProps<Theme>` Sometimes it is necessary and sometimes it is not. I feel confused why this is happening.

Reply:
Interesting, if fails with `flexDirection: 'column',`.

Reply:
I'll create an issue, so that I don't forget to look into it.